amical/tests/test_tools.py

```diff
@@ -104,3 +105,25 @@ def test_compute_pa_niriss_amisim(global_datadir):
         pa = tools.compute_pa(hdr, n_ps)
     assert len(record) == 1
     assert pa == 0.0
+
+
+@pytest.mark.usefixtures("close_figures")
+@pytest.mark.parametrize("list_index_ifu", [[0], [0, 10], [0, 1, 2]])
+@pytest.mark.parametrize("filtname", ["YJ", "YH"])
+def test_get_table_ifu(list_index_ifu, filtname):
+    from matplotlib import pyplot as plt
```

Comment:
nested imports are useful in the application code to reduce startup time, but they don't have much value in test code